tests/test_ses/test_ses_boto3.py

```diff
@@ -79,6 +79,11 @@ def test_send_email():
         **dict(kwargs, Destination={"ToAddresses": too_many_addresses})
     ).should.throw(ClientError)
 
+    invalid_addresses = ["test_to@example.com", "invalid_address"]
```

Comment:
This should be in a separate method, as it is a standalone test case.

Ideally it also uses the with pytest.raises format:
```
with pytest.raises(ClientError) as exc:
    ....
err = exc.value.response["Error"]
err["Message"].should.equal("specific error message")
```
Following this format ensures that the correct error is thrown.

Reply:
Fixed to (separate method + pytest.raises)